.github/workflows/build.yml

```diff
@@ -76,12 +76,9 @@ jobs:
               with:
                   ref: ${{ inputs.gitref }}
 
-            - name: install testmo
```

Comment:
No, I updated the versions for all the Testmo actions and a big part of the latest version is to handle dependencies internally so that all consumers like this project no longer need to install it themselves.